doc/user/radial_and_axial_expansion.rst

```diff
@@ -0,0 +1,86 @@
+*******************************************
+Radial and Axial Expansion and Contraction
+*******************************************
+
+ARMI natively supports both one-dimensional radial and axial expansion. These expansion types function independently of one another and each have their own set of underlying assumptions and use-cases. The remainder of this section is described as follows: in Section :ref:`thermalExpansion` the methodology used for thermal expansion within ARMI is described; in Sections :ref:`radialExpansion` and :ref:`axialExpansion`, we describe the design, limitations, and intended functionality of radial and axial expansion, respecitely.
```

Comment:
"respectively" is misspelled.

I might also suggest to remove the phrase "one-dimensional", just because I find it kinda confusing to read. I don't know whether you mean to apply that to "radial" or to both "radial" and "axial".

Reply:
How about changing "one-dimensional" to linear? The goal here is to show that we aren't doing true volumetric expansion, but rather approximating it via linear expansion.

Reply:
https://github.com/terrapower/armi/pull/1204/commits/656cffbcf8170112b6d8f96d0b2c710d73115ef1

Reply:
Sure, then my suggested rephrase would be something like

"ARMI natively supports linear expansion in both the radial and axial dimensions."

Reply:
I like your suggestion. That's much better. 
https://github.com/terrapower/armi/pull/1204/commits/01ce52ae8e303949f4b776ccd870764f361bceb8